Dockerfile

```diff
@@ -29,18 +29,13 @@ RUN yarn config set network-timeout 300000
 RUN yarn install
 
 # Copy the rest. (node_modules are excluded in .dockerignore)
```

Comment:
Need to update the comments, probably where you put the "copy all" and add a comment about why we do thoses firsts.